.circleci/config.yml

```diff
@@ -62,6 +62,34 @@ jobs:
       - run: docker login -u="${DOCKER_USERNAME}" -p="${DOCKER_PASSWORD}"
       - run: make docker-push
 
+  publish_release:
+    docker:
+      # Available from https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/improbable-eng/thanos
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.07.0-ce
+      - attach_workspace:
+          at: .
+      - run: make promu
+      - run: promu crossbuild tarballs
+      - run: promu checksum .tarballs
```

Comment:
can we have make file commad for all of these instead? That depends on `$(promu)`

Reply:
Yea, I guess we can do it that way.